pkg/config/linters_settings.go

```diff
@@ -909,11 +909,11 @@ type UseStdlibVarsSettings struct {
 	TimeLayout         bool `mapstructure:"time-layout"`
 	CryptoHash         bool `mapstructure:"crypto-hash"`
 	DefaultRPCPath     bool `mapstructure:"default-rpc-path"`
-	OSDevNull          bool `mapstructure:"os-dev-null"`
+	OSDevNull          bool `mapstructure:"os-dev-null"` // Deprecated
```

Comment:
don't see `usestdlibvar` config in reference files 🤔

Reply:
`usestdlibvars` is in the reference files:

https://github.com/golangci/golangci-lint/blob/55b2f5ddebb8270a2dac581cd072fbf287bd3174/.golangci.reference.yml#L2612

https://github.com/golangci/golangci-lint/blob/55b2f5ddebb8270a2dac581cd072fbf287bd3174/.golangci.reference.yml#L2261

https://github.com/golangci/golangci-lint/blob/55b2f5ddebb8270a2dac581cd072fbf287bd3174/.golangci.next.reference.yml#L2740

https://github.com/golangci/golangci-lint/blob/55b2f5ddebb8270a2dac581cd072fbf287bd3174/.golangci.next.reference.yml#L2272